ui/chat/src/spam.ts

```diff
@@ -39,6 +39,10 @@ const spamRegex = new RegExp(
     'qps.ru',
     'tiny.cc/',
     'trasderk.blogspot.com',
+    't.ly/',
+    'shorturl.at/'
+
+    
```

Comment:
Delete the 2 extra lines of separation, add a comma at the end lines

Reply:
Thanks, I just tried, the checks are still running.
Thanks for the hint.